packages/perseus/src/widgets/interactive-graphs/interactive-graph.testdata.ts

```diff
@@ -1014,6 +1017,7 @@ export const staticGraphQuestionWithAnotherWidget: () => PerseusRenderer =
 
 export const interactiveGraphWithAriaLabel: PerseusRenderer =
     interactiveGraphQuestionBuilder()
+        .withNoInteractiveFigure()
```

Comment:
Updating this so that the test for full graph aria labels can focus on just the graph without any interactive elements.